packages/flutter_tools/lib/src/windows/visual_studio.dart

```diff
@@ -163,14 +192,20 @@ class VisualStudio {
   Map<String, dynamic> get _usableVisualStudioDetails {
     _cachedUsableVisualStudioDetails ??=
         _visualStudioDetails(requiredComponents: _requiredComponents().keys);
+    // If a stable version is not found, try searching for a pre-release version.
+    _cachedUsableVisualStudioDetails ??= _visualStudioDetails(
+        requiredComponents: _requiredComponents().keys,
+        additionalArguments: <String>['-prerelease']);
```

Comment:
Nit: maybe make `-prerelease` an internal string constant since it's used twice.

Reply:
Done